dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScopeTest.kt

```diff
@@ -741,8 +823,60 @@ internal class RumSessionScopeTest {
         verify(mockDevLogHandler).handleLog(Log.WARN, RumSessionScope.MESSAGE_MISSING_VIEW)
     }
 
-    private fun Forge.forgeValidBackgroundEvent(): RumRawEvent {
-        val fakeEventTime = Time()
+    @Test
+    fun `M send warn dev log W handleEvent { no active scope, app launch event not relevant}`(
+        forge: Forge
+    ) {
+        // GIVEN
+        testedScope.applicationDisplayed = false
+        val fakeEvent = forge.forgeInvalidAppLaunchEvent()
+
+        // WHEN
+        testedScope.handleEvent(fakeEvent, mockWriter)
+
+        // THEN
+        verify(mockDevLogHandler).handleLog(Log.WARN, RumSessionScope.MESSAGE_MISSING_VIEW)
+    }
+
+    private fun Forge.forgeValidBackgroundEvent(
+        eventTime: Time? = null
+    ): RumRawEvent {
+        val fakeEventTime = eventTime ?: Time()
+        val fakeName = this.anAlphabeticalString()
+
+        return this.anElementFrom(
+            listOf(
+                RumRawEvent.StartAction(
+                    this.aValueFrom(RumActionType::class.java),
+                    fakeName,
+                    this.aBool(),
+                    emptyMap(),
+                    fakeEventTime
+                ),
+                RumRawEvent.AddError(
+                    fakeName,
+                    this.aValueFrom(RumErrorSource::class.java),
+                    stacktrace = null,
+                    throwable = null,
+                    isFatal = this.aBool(),
+                    attributes = emptyMap(),
+                    eventTime = fakeEventTime
+                ),
+                RumRawEvent.StartAction(
+                    this.aValueFrom(RumActionType::class.java),
+                    fakeName,
+                    this.aBool(),
+                    emptyMap(),
+                    fakeEventTime
+                )
+            )
+        )
+    }
+
+    private fun Forge.forgeValidAppLaunchEvent(
+        eventTime: Time? = null
+    ): RumRawEvent {
+        val fakeEventTime = eventTime ?: Time()
         val fakeName = this.anAlphabeticalString()
 
         return this.anElementFrom(
```

Comment:
I cannot put a comment on the valid line (github doesn't let me put it outside of the diff context), but this list contains `StartAction` twice, it should be a `StartResource` in the latter case.